src/NzbDrone.Core/Download/Clients/Vuze/Vuze.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Download.Clients.Transmission;
+using NzbDrone.Core.MediaFiles.TorrentInfo;
+using NzbDrone.Core.RemotePathMappings;
+
+namespace NzbDrone.Core.Download.Clients.Vuze
+{
+    public class Vuze : Transmission.Transmission
+    {
+        private const int MINIMAL_SUPPORTED_PROTOCOL_VERSION = 14;
+        private readonly ITransmissionProxy _proxy;
+
+        public Vuze(ITransmissionProxy proxy,
+                    ITorrentFileInfoReader torrentFileInfoReader,
+                    IHttpClient httpClient,
+                    IConfigService configService,
+                    IDiskProvider diskProvider,
+                    IRemotePathMappingService remotePathMappingService,
+                    Logger logger)
+            : base(proxy, torrentFileInfoReader, httpClient, configService, diskProvider, remotePathMappingService, logger)
+        {
+            _proxy = proxy;
+        }
+
+        protected override DownloadClientItem CreateDownloadItem(TransmissionTorrent torrent, OsPath outputPath)
+        {
+            // Vuze reports outputPath containing the torrent name already, so we need to subtract it
+            // for the Transmission protocol compatibility
+
+            if (outputPath.FullPath.EndsWith(torrent.Name, StringComparison.Ordinal))
+            {
+                var startOfTorrentName = outputPath.FullPath.LastIndexOf(torrent.Name, StringComparison.Ordinal);
+                outputPath = new OsPath(outputPath.FullPath.Substring(0, startOfTorrentName));
```

Comment:
This will be dead, but for what its worth you could do:

`outputPath = new OsPath(outputPath.FullPath.Substring(0, outputPath.FullPath.Length - torrent.Name .Length));`